docs/UserDefinedLanguageServer.md

```diff
@@ -87,14 +87,46 @@ The `Debug tab` is available when you have created the language server definitio
 level Trace used in LSP console.
 
 ### Using template
+Template can be used to quickly create user defined language server pre-filled 
+with server name, command, mappings and potential configurations.
 
-The `Template combo-box` provides some `language servers templates`, pre-filled with server name, command, mappings and potential configuration. 
+#### Default template
+The `Template combo-box` provides some default language servers templates (located in templates directory classpath), 
+pre-filled with server name, command, mappings and potential configuration.
 
-![New Language Server with Template](./images/user-defined-ls/NewLanguageServerWithTemplate.png)
+If the template folder contains a `README.md` file, you can press the help button to open the instructions.
 
- * [Clojure LSP](./user-defined-ls/clojure-lsp.md)
- * [CSS Language Server](./user-defined-ls/vscode-css-language-server.md)
- * [Go Language Server](./user-defined-ls/gopls.md) 
- * [HTML Language Server](./user-defined-ls/vscode-html-language-server.md)
- * [Scala Language Server (Metals)](./user-defined-ls/metals.md)
- * [TypeScript Language Server](./user-defined-ls/typescript-language-server.md)
+![New Language Server with Default Template](./images/user-defined-ls/NewLanguageServerWithDefaultTemplate.png)
+
+* [Clojure LSP](./user-defined-ls/clojure-lsp.md)
+* [CSS Language Server](./user-defined-ls/vscode-css-language-server.md)
+* [Go Language Server](./user-defined-ls/gopls.md)
+* [HTML Language Server](./user-defined-ls/vscode-html-language-server.md)
+* [Scala Language Server (Metals)](./user-defined-ls/metals.md)
+* [TypeScript Language Server](./user-defined-ls/typescript-language-server.md)
+
+#### Custom template
+
+The `Template file-selector` can be used to select a directory from the file system to load a custom language server template, 

```

Comment:
The `Add new template...` combo item from the `Template combo-box` can be used to select a...
... mappings and potential configuration.


Add a screenshot by drawing a red square to highlight the `Add new template...`